### DISCUSSION TITLE
Slow in Stimulus Presentation

### DISCUSSION THREAD

**Discussion by billycysung** [jspsych/jsPsych#218] [Q&A] [answered]
Hi,

I am new to jsPsyc and also new to Javascript. I have been trying to code an quick reaction time task and attached is my code. The problem I am having is, when I present the block "HinF" (essentially a 14kb image named "HinF.jpeg") after the block "cross" (a fixation cross), the image takes 4 seconds to appear.

However, when I present the same image the first time in the block "Practice", the image comes up immediately. I am not sure why I am getting a lag like this when I present it the second time. Any help will be greatly appreciated!

Cheers,
Billy

[try.docx](https://github.com/jodeleeuw/jsPsych/files/83686/try.docx)

**Comment by jodeleeuw** [comment, marked as answer]
It might be related to the `timing_post_trial` of the previous trial. The default value is to put a 1s blank screen between all trials. You can override this by setting `timing_post_trial` to 0.

**Comment by billycysung** [comment]
That sorted the problem! Thanks so much!
